app/src/main/java/com/rkm/tasky/navigation/AppNavigation.kt

```diff
@@ -87,3 +79,29 @@ fun AppNavigation(
         }
     }
 }
+
+sealed class Destination {
+    @Serializable
+    data object Authentication : Destination()
+
+    @Serializable
+    object Login : Destination()
+
+    @Serializable
+    object Registration : Destination()
+
+    @Serializable
+    object Home : Destination()
+
+    @Serializable
+    object Agenda : Destination()
```

Comment:
Minor - these can also be data object to take advantage of the default override of the `toString()` function